### PR TITLE
master-lps-102636_1

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-102636

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+		SitePages.gotoPageMenuItem(
+			menuItem = "Configure",
+			pageName = "Test Page Name");
+
+		Navigator.gotoNavTab(navTab = "SEO");
+
+		Panel.expandPanel(panel = "SEO");
+
+		AssertElementPresent(
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase]: https://github.com/austinchiang/liferay-portal/pull/553/commits/fce5726140aba313dd086ea4c693ba74b79ed237#diff-575f61f294c55166be8b08876ed47a83R104-R106 only asserts the header. We need to assert the fields are there by default too. Please use a macro called OpenGraph.viewDefaultConfiguraton for this.

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		}
+	}
+
+	@description = "This ensures that OG tags will not appear when Open Graph is disabled."
+	@priority = "5"
+	test DisableConfiguration {
+		property test.name.skip.portal.instance = "OpenGraph#DisableConfiguration";
+		var portalURL = PropsUtil.get("portal.url");
+
+		SitePages.openPagesAdmin(siteURLKey = "test-site-name");
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase]: https://github.com/austinchiang/liferay-portal/pull/553/commits/fce5726140aba313dd086ea4c693ba74b79ed237#diff-575f61f294c55166be8b08876ed47a83R56-R68 This is not in the test steps and is covered by Smoke. Please remove

**Comment by austinchiang** [comment]
Please resend

**Pull request closed without merging**